### PR TITLE
fix: SSL Query for API Gateway

### PR DESCRIPTION
**Pull request by bbernays**

### CODE REVIEW

File: tools/find-sensitive-strings.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
+package main
```
Comment (shimonp21) [inline tools/find-sensitive-strings.go:1]: Add comment on what this tool does?

File: tools/find-sensitive-strings.go [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+		if !accountAllowed {
+			validatedMatches = append(validatedMatches, fmt.Sprintf("%s found at index %d, in %s", match, i, fileName))
+		}
+	}
+	return validatedMatches, nil
+}
+
+func checkFile(filePath string) ([]string, error) {
+	var re = regexp.MustCompile(`(?m)\d{12}`)
```
Comment (shimonp21) [inline tools/find-sensitive-strings.go:49]: Add comment to explain what this regex does?

**Review by shimonp21 — APPROVED** [review]

File: tools/find-sensitive-strings.go (hunk 1 above)
Comment (bbernays) [reply tools/find-sensitive-strings.go:1]: fixed

Comment (bbernays) [reply tools/find-sensitive-strings.go:1]: Also I am writing more documentation in the docs repo about snapshot testing

File: tools/find-sensitive-strings.go (hunk 2 above)
Comment (bbernays) [reply tools/find-sensitive-strings.go:54]: done

**Pull request merged**